src/components/scenes/PromotionSettingsScene.tsx

```diff
@@ -80,19 +81,6 @@ export class PromotionSettingsComponent extends React.Component<Props> {
   }
 }
 
-const getStyles = cacheStyles((theme: Theme) => ({
-  textBlock: {
-    backgroundColor: theme.settingsRowBackground,
-    padding: theme.rem(0.5)
-  },
-  textRow: {
-    color: theme.primaryText,
-    fontFamily: theme.fontFaceDefault,
-    fontSize: theme.rem(1),
-    margin: theme.rem(0.5)
-  }
-}))
```

Comment:
Margin/padding wont match the same presentation as in `SettingsSubHeader` from what I can tell. Is this intentional?

Reply:
Yes, the three lines are more spaced out after this change. I think it looks better this way, especially because it's more consistent with the other settings scenes. I think this is part of what Jon was talking about - removing special cases by creating common components.